account_financial_risk/i18n/es.po

```diff
@@ -423,3 +424,29 @@ msgstr "account.config.settings"
 #: model:ir.model,name:account_financial_risk.model_partner_risk_exceeded_wiz
 msgid "partner.risk.exceeded.wiz"
 msgstr "partner.risk.exceeded.wiz"
+
+#. module: account_financial_risk
+#: model:ir.model,name:account_financial_risk.model_res_config_settings
+msgid "res.config.settings"
+msgstr ""
+
+#~ msgid "Continue method"
```

Comment:
Please don't use POEdit or configure it properly for not letting this residual lines (and check also line wrapping).

Reply:
Changes done